Rakefile

```diff
@@ -137,7 +137,7 @@ beaker_node_sets.each do |node|
     "beaker:#{node}:acceptance", [:version, :filter] => [:spec_prep]
   ) do |task, args|
     ENV['BEAKER_set'] = node
-    args.with_defaults(:version => '6.2.3', :filter => nil)
+    args.with_defaults(:version => '6.8.6', :filter => nil)
```

Comment:
any reason to not use 6.8.7?

Reply:
None other than `6.8.7` wasn't available when I did this initial commit... Will update in future round :)